android/src/main/java/org/reactnative/camera/RNCameraViewHelper.java

```diff
@@ -310,24 +310,33 @@ public static WritableMap getExifData(ExifInterface exifInterface) {
   }
 
   public static void setExifData(ExifInterface exifInterface, WritableMap exifMap) {
-    ReadableMapKeySetIterator iterator = exifMap.keySetIterator();
-    while (iterator.hasNextKey()) {
-      String key = iterator.nextKey();
-      switch (exifMap.getType(key)) {
-        case Null:
-          exifInterface.setAttribute(key, null);
-          break;
-        case Boolean:
-          exifInterface.setAttribute(key, Boolean.toString(exifMap.getBoolean(key)));
-          break;
-        case Number:
-          exifInterface.setAttribute(key, Double.toString(exifMap.getDouble(key)));
-          break;
-        case String:
-          exifInterface.setAttribute(key, exifMap.getString(key));
-          break;
+    for (String[] tagInfo : exifTags) {
+      String name = tagInfo[1];
+      if (exifMap.hasKey(name)) {
+        String type = tagInfo[0];
+        switch (type) {
```

Comment:
It seems as though your switch statement is missing a case for "boolean".  I'm not sure if that's a valid EXIF tag type, but the previous functionality had a special case for extracting boolean data: `Boolean.toString(exifMap.getBoolean(key))`  Should this be included somewhere in the new code?

Reply:
The logic is the same as in getExifData at https://github.com/react-native-community/react-native-camera/blob/b1791b45ccbde034f44cc91f1e515bdc3292750d/android/src/main/java/org/reactnative/camera/RNCameraViewHelper.java#L282-L299

There, a list of EXIF tags is used:
https://github.com/react-native-community/react-native-camera/blob/b1791b45ccbde034f44cc91f1e515bdc3292750d/android/src/main/java/org/reactnative/camera/RNCameraViewHelper.java#L28-L32

So, as boolean values will never be read there, and the result of getExifData is used for setExifData, I believe this covers all cases.